### ISSUE TITLE
Exit Code of Remote Script

### ISSUE THREAD

**Issue by wtlinnertz** [WinRb/WinRM#222]
Using the example below from the read.me, how would you capture the exit code from a remote command shell?

I need to exit the Ruby script with the same exit code as the remote shell.

```
require 'winrm'
opts = { 
  endpoint: 'http://myhost:5985/wsman',
  user: 'administrator',
  password: 'Pass@word1'
}
conn = WinRM::Connection.new(opts)
conn.shell(:powershell) do |shell|
  shell.run('$PSVersionTable') do |stdout, stderr|
    STDOUT.print stdout
    STDERR.print stderr
  end
end
```

Thanks,
Todd

**Comment by sneal** [comment]
The `shell.run` method returns a [WinRM::Output](https://github.com/WinRb/WinRM/blob/master/lib/winrm/output.rb#L25) object which has an exitcode property.

Here's an example that _should_ work (I didn't try it).

```ruby
conn.shell(:powershell) do |shell|
  output = shell.run('exit 1') do |stdout, stderr|
    STDOUT.print stdout
    STDERR.print stderr
  end
  puts "The script exited with exit code #{output.exitcode}"
end
```

We should update the readme to show this in the example.

**Comment by wtlinnertz** [comment]
The ```exitcode``` property worked great.  Thanks for the help.

Todd

**Comment by mwrock** [comment]
fixed via #224
